src/bindings/cpp/include/keyset.hpp

```diff
@@ -31,7 +31,9 @@ class KeySet
 	inline KeySet(ckdb::KeySet *k);
 	inline KeySet(const KeySet &other);
 
-	inline explicit KeySet(size_t alloc, va_list ap);
+#if defined(__amd64__) || defined(__x86_64__) || defined(__i386__)
```

Comment:
So e.g. `inline explicit KeySet(int unused, size_t alloc, va_list va)` is what you prefer?

I think the va_list-constructor won't be used much. Bindings are unable to use it. They are even unable to use the variadic arguments constructor. So right now we don't use it all.